README.md

```diff
@@ -6,4 +6,93 @@ Model Independent Chemical Mechanisms.
 [![License](https://img.shields.io/github/license/NCAR/micm.svg)](https://github.com/NCAR/micm/blob/master/LICENSE)
 [![CI Status](https://github.com/NCAR/micm/actions/workflows/test.yml/badge.svg)](https://github.com/NCAR/micm/actions/workflows/test.yml)
 
+
+# Building and installing
+To build and install MICM locally, you must have the following libraries installed:
+
+- [json-fortran](https://github.com/jacobwilliams/json-fortran)
+
+- [sphinx](https://github.com/sphinx-doc/sphinx)
+- [sphinx-book-theme](https://github.com/executablebooks/sphinx-book-theme)
+- [sphinx-design](https://github.com/executablebooks/sphinx-design)
+- [breathe](https://github.com/breathe-doc/breathe)
+
+You must also have CMake installed on your machine. 
+
+To install MICM locally,
+open a terminal window, navigate to a folder where you would like the MICM files to exist,
+and run the following commands:
+
+## Build and run (Docker version)
+
+To build and run the stand-alone version of MICM, you must have [Docker Desktop](https://www.docker.com/get-started) installed and running. With Docker Desktop running, open a terminal window and run the following command to start the MICM container:
+
+```
+docker run -it ghcr.io/ncar/mcim:release bash
+```
+
+Inside the container, you can run the MICM tests from the `/build/` folder:
+
+```
+cd build/
+# to run the tests
+make test
+# to use the standalone tool
+./micm examples/full_config.json
```

Comment:
MICM does not have an executable standalone tool. Please remove this line and the comment